docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -225,38 +215,47 @@ CEPH TEST SUMMARY: Suite contains 812, ran 387 tests, Passed: 387, Skipped: 0, F
 Following the 'Run a Single Ceph S3 Test' steps.
 
 ### 1) Test Pass
-For example: `s3tests_boto3.functional.test_s3:test_basic_key_count`
+For example: `s3tests_boto3/functional/test_s3.py::test_basic_key_count`
 ```
-bash-4.4$ S3TEST_CONF=src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf ./src/test/system_tests/ceph_s3_tests/s3-tests/virtualenv/bin/nosetests s3tests_boto3.functional.test_s3:test_basic_key_count
-.
-----------------------------------------------------------------------
-Ran 1 test in 4.914s
+bash-4.4$ S3TEST_CONF=$PWD/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf tox -c src/test/system_tests/ceph_s3_tests/s3-tests/tox.ini -- --disable-pytest-warnings $PWD/src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py::test_account_usage
+
+...
+
+====================================================================================== 1 passed, 1 warning in 15.71s =======================================================================================
+_________________________________________________________________________________________________ summary __________________________________________________________________________________________________
+  py: commands succeeded
+  congratulations :)
+
 
-OK
 ```
 ### 2) Test Fail
-For example: `s3tests_boto3.functional.test_s3:test_account_usage`
+For example: `s3tests_boto3/functional/test_s3::test_account_usage.py`
 
 ```
-bash-4.4$ S3TEST_CONF=src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf ./src/test/system_tests/ceph_s3_tests/s3-tests/virtualenv/bin/nosetests s3tests_boto3.functional.test_s3:test_account_usage
-E
-======================================================================
-ERROR: s3tests_boto3.functional.test_s3.test_account_usage
-----------------------------------------------------------------------
-Traceback (most recent call last):
-  File "/root/node_modules/noobaa-core/src/test/system_tests/ceph_s3_tests/s3-tests/virtualenv/lib/python3.6/site-packages/nose/case.py", line 198, in runTest
-    self.test(*self.arg)
-  File "/root/node_modules/noobaa-core/src/test/system_tests/ceph_s3_tests/s3-tests/s3tests_boto3/functional/test_s3.py", line 1325, in test_account_usage
-    summary = parsed['Summary']
-KeyError: 'Summary'
--------------------- >> begin captured logging << --------------------
```

Comment:
In the new setup, don't we have a log dump/error traceback?

Reply:
we have, it now shows the entire code, and where it fails. so I didn't add to not make it too long. but if I add an image it should be fine